src/usr/local/www/system_advanced_admin.php

```diff
@@ -252,15 +259,24 @@
 			unset($config['system']['ssh']['sshdkeyonly']);
 		}
 
+		$sshd_agentforwarding = isset($config['system']['ssh']['sshdagentforwarding']);
+		if ($_POST['sshdagentforwarding']) {
+			$config['system']['ssh']['sshdagentforwarding'] = true;
```

Comment:
For consistency this should be ` = "enabled";`

Reply:
Right!